include/field/field_system.h

```diff
@@ -54,7 +54,7 @@ typedef struct FieldSystem_t {
     int unk_18;
     Location * location;
     int unk_20;
-    UnkStruct_020203AC * unk_24;
+    UnkStruct_020203AC * camera;
```

Comment:
Because various encounter effects make use of it and modify its parameters. I tried messing around with the interpolation values and saw a clear difference in how the camera behaves. I.e. changing the distance it moves away from the player.